styles/about_me.css

```diff
@@ -0,0 +1,55 @@
+
+body {
+  position: absolute;
+  background-size: cover;
+  font-family: 'Open Sans Condensed', sans-serif;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+}
+
+/*space these words out more*/
+.site-nav {
+  float: right;
+  padding: 10px 15px 0 0;
+  color: white;
+  font-size: 26px;
+}
+
+a:hover {
+  color:black;
+}
+
+img {
+  min-height: 100%;
+  min-width: 1024px;
+  width: 100%;
+  height: auto;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.about-me {
+  position: relative;
+  margin: 150px 0 0 80px;
+  width: 40%;
+  font-size: 25px;
+  color: white;
+}
+.about-me p {
```

Comment:
Sometimes relying on the type of element used for CSS styles can bite you - think about what would happen if someone came back later and asked you to change all the `<p>` elements to `<divs>` (or something).  You would need to remember to also update your CSS! And it might take you a while to find that bug!  A class name on the p-tag would prevent that.  That's just something to think about -- styling a child element is not wrong.